Documentation/running-docker-images.md

```diff
@@ -6,14 +6,14 @@ rkt features native support for fetching and running Docker container images.
 
 To reference a Docker image, use the `docker://` prefix when fetching or running images.
 
-Note that Docker images do not support signature verification, and hence it's necessary to use the `--insecure-skip-verify` flag.
+Note that Docker images do not support signature verification, and hence it's necessary to use the `--insecure-options=image` flag.
 
 As a simple example, let's run the latest `redis` container image from the default Docker registry:
 
 ```
-# rkt --insecure-skip-verify run docker://redis
+# rkt --insecure-options=image run docker://redis
 rkt: fetching image from docker://redis
-rkt: warning: TLS verification and signature verification has been disabled
+rkt: warning: Image signature verification has been disabled
```

Comment:
`Image` is in lowercase:
```
rkt/images.go:339: stderr("rkt: warning: image signature verification has been disabled")
```
So the documentation should be fixed accordingly. (There are other occurrences too)